src/cpp/session/modules/SessionFind.cpp

```diff
@@ -1786,7 +1783,7 @@ core::Error Replacer::replaceRegexWithCase(size_t matchOn, size_t matchOff,
 {
    try
    {
-      boost::regex_constants::syntax_option_type flags = boost::regex::egrep & boost::regex::bk_vbar;
+      boost::regex_constants::syntax_option_type flags = boost::regex::perl;
       boost::regex find(findRegex, flags);
```

Comment:
```suggestion
      boost::regex find(findRegex);
```
Since perl is the default we don't need to pass it.